src/server/bitops_family.cc

```diff
@@ -0,0 +1,267 @@
+#include "server/bitops_family.h"
+
+#include <bitset>
+
+#include "server/string_family.h"
+extern "C" {
+#include "redis/object.h"
+}
+
+#include "base/logging.h"
+#include "server/command_registry.h"
+#include "server/common.h"
+#include "server/conn_context.h"
+#include "server/engine_shard_set.h"
+#include "server/error.h"
+#include "server/tiered_storage.h"
+#include "server/transaction.h"
+#include "util/varz.h"
+
+namespace dfly {
+using namespace facade;
+
+namespace {
+static const int32_t OFFSET_FACTOR = 8;  // number of bits in byte
+static const long INDEX_MAPPING[] = {    // or same reason at REDIS the indenes at left to right
```

Comment:
For same reason at REDIS the indenes are from left to right

Reply:
this was removed